apps/src/blockly/themes/cdoDark.js

```diff
@@ -9,9 +9,9 @@ export default GoogleBlockly.Theme.defineTheme(Themes.DARK, {
   base: DarkTheme,
   blockStyles: cdoBlockStyles,
   componentStyles: {
-    workspaceBackgroundColour: color.neutral_dark90,
-    blackBackground: color.neutral_dark80,
-    flyoutBackgroundColour: color.neutral_dark70,
+    workspaceBackgroundColour: color.neutral_dark,
+    blackBackground: color.neutral_dark90,
+    flyoutBackgroundColour: color.neutral_dark80,
```

Comment:
This theme is used across labs. I’m sure it’s fine, but could we get a quick screenshot of how this looks in a Dance Party project or another lab? (Dark mode can be enabled by right-clicking the workspace)

Reply:
Good call.  What do you think?  We might need to make the header bar darker?

<img width="1145" alt="Screenshot 2023-05-16 at 10 21 21 AM" src="https://github.com/code-dot-org/code-dot-org/assets/2205926/82ad64b0-99fb-4d91-98f6-58ff3ca637a0">

Reply:
Yeah exactly. But I think the rebrand work is what set the current header color and then @levadadenys had to lighten the workspace to bring back the contrast.

Reply:
Here's the latest.  The PR description screenshot has been updated too.
<img width="1195" alt="Screenshot 2023-05-17 at 10 07 37 AM" src="https://github.com/code-dot-org/code-dot-org/assets/2205926/cbea4281-68fe-40a9-8938-9f769d934b1a">

Reply:
A noticeable improvement! Thanks for iterating on this with our design team.

Reply:
The PR description now contains the latest approach.